android/app/src/main/AndroidManifest.xml

```diff
@@ -57,8 +57,6 @@
                 android:name="com.reactnativenavigation.controllers.NavigationActivity"
                 android:configChanges="keyboard|keyboardHidden|orientation|screenSize"/>
         <activity
-                android:noHistory="true"
-                android:excludeFromRecents="true"
```

Comment:
What is the reason for removing this from the Manifest?

Theses properties are fit for what the SharedExtension experience should be like:

**noHistory:**
>Whether or not the activity should be removed from the activity stack and finished (its finish() method called) when the user navigates away from it and it's no longer visible on screen — "true" if it should be finished, and "false" if not. The default value is "false". 

**excludeFromRecents:**
> Whether or not the task initiated by this activity should be excluded from the list of recently used applications, the overview screen. That is, when this activity is the root activity of a new task, this attribute determines whether the task should not appear in the list of recent apps. Set "true" if the task should be excluded from the list; set "false" if it should be included. The default value is "false". 

Did these properties class with the cold startup process?

Reply:
If we keep this two the extension gets dismissed when the EMM authentication gets dismissed and then starts behaving very erratic (this only happens if EMM is enabled)